CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # Change Log
 
+## [v1.8.0](https://github.com/chef/knife-azure/tree/v1.8.0) (2017-mm-dd)
+
+[Full Changelog](https://github.com/chef/knife-azure/compare/v1.7.0...v1.8.0)
+
+**Closed issues:**
+- `--delete-resource-group` works, but exits as failed [\#458](https://github.com/chef/knife-azure/issues/458)
+- Master is broken for any azurerm server create operations that involve virtual networks [\#455](https://github.com/chef/knife-azure/issues/455)
+- Running `azurerm server create` with Chef Environment argument does not correctly assign environment to the created node [\#454](https://github.com/chef/knife-azure/issues/454)
+- Can not create vms in required availability set (ARM) [\#446](https://github.com/chef/knife-azure/issues/446)
+- Bootstrapping in Azurerm is not happenning fully. [\#449](https://github.com/chef/knife-azure/issues/449)
+- Auto-Generated Storage Account Name can produce an Exception [\#434](https://github.com/chef/knife-azure/issues/434)
+- node_name in client.rb always has a '0' appended resulting in auth error [\#439](https://github.com/chef/knife-azure/issues/439)
+- Knife Azure does not work with berkshelf [\#423](https://github.com/chef/knife-azure/issues/423)
+- --node-ssl-verify-mode none' does not write appropriate value to resulting client.rb [\#435](https://github.com/chef/knife-azure/issues/435)
+- knife azure server delete doesn't delete servers joined to a domain [\#258](https://github.com/chef/knife-azure/issues/258)
```

Comment:
https://github.com/chef/knife-azure/issues/440 is also fixed in the release. Please add that.